registry.json

```diff
@@ -3203,6 +3203,18 @@
  "telegram": "@ADABankAU",                                             
  "operatorName": "",                                   
  "github": "OffTheRicta"                                                     
-}
-
+},
+  {
+      "ticker":"APEX",
+      "poolId":"538299a358e79a289c8de779f8cd09dd6a6bb286de717d1f744bb357",
+      "social":{
+         "twitter":"cardanoapexpool",
+         "website":"https://apexpool.info",
+         "reddit":"georgem1976",
+         "youtube":""
```

Comment:
Please add instagram and facebook (even if blank). I think they are required field.